src/unlinkcommand.js

```diff
@@ -14,6 +14,27 @@ import findLinkRange from './findlinkrange.js';
  */
 export default class UnlinkCommand extends Command {
 	/**
+	 * @see core.command.Command
+	 * @param {core.editor.Editor} editor
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		/**
+		 * Flag indicating whether command is active. For collapsed selection it means that typed characters will have
+		 * the command's attribute set. For range selection it means that all nodes inside have the attribute applied.
+		 *
+		 * @observable
+		 * @member {Boolean} core.command.ToggleAttributeCommand#value
+		 */
+		this.set( 'hasValue', undefined );
```

Comment:
> HOWEVER, why is this property called hasValue and not simply value? Does it represent something else?

Is `hasValue` because I use `hasAttribute()` instead `getAttribute()`, but if `value` is preferred I'll change it.

> BTW. just a thing which came to my mind – similarly to the idea with an image's value we should remember that a link may have more attributes than just its href. I don't know how it's now implemented, but I guess that we can make its value an object already.

I was thinking about it and I've decided to go simplest way and refactor it in time when we will be adding more attributes.